platform-sdk/docs/core/dynamic-address-book/roster/glossary-of-terms.md

```diff
@@ -0,0 +1,100 @@
+# Roster Glossary of Terms
+![Event Window Terminology](./EventWindowTerminology.drawio.svg)
```

Comment:
Great diagram. My only suggestion on it is to label the left and right sides at the top. The left side is all about rounds, and the right side is all about the roster associated with the round on the left.